### PR TITLE
Add Right margin plugin to build script file

### PR DESCRIPTION
**Pull request by alexander-aleksandrov**
Update version of tools projects and add right margin plugin

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -38,7 +38,7 @@ buildscript {
         //TODO:2016-12-12:alexander.yevsyukov: Advance the plug-in version together with all
         // the components and change the version of the dependency below to
         // `spineVersion` defined above.
-        spineToolsVersion = '0.9.1-SNAPSHOT'
+        spineToolsVersion = '0.9.2-SNAPSHOT'
```
Comment (dmdashenkov) [inline build.gradle]: It seems like this version number is already taken.
Try `0.9.4-SNAPSHOT`.

File: build.gradle [hunk 2]
```diff
@@ -140,10 +140,16 @@ subprojects {
...
-
             threshold = 0
             reportType = "warn"
         }
+
+        // Right margin line checker.
+        //
+        // Finds and reports about the lines of a code with length above threshold.
+        rightMarginWrappingChecker {
+            threshold = 100
```
Comment (dmdashenkov) [inline build.gradle]: Why do we still call this parameter `threshold`? Let's make it consistent with the tools internal naming.

Comment (alexander-aleksandrov) [inline build.gradle]: Done [here](https://github.com/SpineEventEngine/tools/pull/88)

Comment (alexander-aleksandrov) [inline build.gradle]: We decided on a meeting to name this parameter `maxTextWidth`

**Comment by alexander-aleksandrov** [comment]
@dmdashenkov PTAL

**Pull request merged**